sites/org/package.json

```diff
@@ -16,7 +16,6 @@
   "scripts": {
     "build": "next build",
     "cibuild": "yarn build",
-    "clean": "rimraf pages/*.mjs && rimraf pages/*/*.mjs && rimraf pages/v/*/*.mjs",
```

Comment:
This needs to be changed in `config/scripts.yml` as well. I also think maybe we should talk about what we do and don't clean... I think that for instance this could be changed to `rimraf prebuild/* public/locales/*/* public/feeds/*` to great effect, but I'm not sure what we are and aren't aiming to clean

Reply:
This is a good catch by @BenJamesBen but as @eriese pointed out, the real fix is to update the scripts config. If not, this will just get overwritten next time `yarn reconfigure` runs